music21/abcFormat/testFiles.py

```diff
@@ -612,8 +676,116 @@ def testBasic(self):
             if title == 'D Fragment':
                 sharps = s.parts[0].keySignature.sharps
                 self.assertEqual(sharps, 7, 'C# key signature should be parsed as 7 sharps')
+                
+    def testAbc21(self):
+        from music21 import abcFormat, note
+        from music21.abcFormat import translate
+        from music21.musicxml import m21ToXml
 
+        af = abcFormat.ABCFile(abcVersion="2.1")
+
+        gex = m21ToXml.GeneralObjectExporter()
         
+        files = [carryThrough]
```

Comment:
If we're only doing one file, go ahead and read it without the for loop, which was confusing.

Reply:
Okay, I will add more test methods whenever I deal with ties and chords.